### ISSUE TITLE
Production build gives warnings which results into application failure on the server.

### ISSUE THREAD

**Issue by ahaan729** [angular/angular-cli#3016]
### OS?
>Windows 10

### Versions.
> angular-cli: 1.0.0-beta.19-3
node: 7.0.0
os: win32 x64

### Repro steps.
>ng build -p --bh /ng-forms/

### The log given by the failure.
> WARNING in styles.3dd438f66d18a543d9d3.bundle.js from UglifyJs
Condition always false [C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/style-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/css-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/postcss-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/bootstrap/dist/css/bootstrap.css:10,0]
Dropping unreachable code [C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/style-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/css-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/postcss-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/bootstrap/dist/css/bootstrap.css:12,0]
Side effects in initialization of unused variable update [C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/style-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/css-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/postcss-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/bootstrap/dist/css/bootstrap.css:7,0]
Condition always false [C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/style-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/css-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/postcss-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/src/styles.css:10,0]
Dropping unreachable code [C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/style-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/css-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/postcss-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/src/styles.css:12,0]
Side effects in initialization of unused variable update [C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/style-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/css-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/\~\/postcss-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/src/styles.css:7,0]

### Mention any other details that might be useful.
I have posted the warnings log only as the rest of them were about the dropping unused functions.
Because of the above warnings, when I deploy the application on the server the .js files doesn't loads.

But the same works when I build the application in the development mode without any warnings. Below command works fine. 
>ng build -d --bh /ng-forms/

I am clueless why it is not working in the production mode. Am I doing something wrong or is there some issue with the production build?

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by kesuskim** [comment]
Not sure but how about using Node.js LTS version, 6.9.1? When I saw some log messages in my experience, I saw deprecation warnings that some will not work in Node higher than 7.

**Comment by ahaan729** [comment]
I am using node.js Current v7.0.0
I think you are referring to the warning messages showing deprecation of graceful, loadsh, etc. In that case it asks to update those packages and I have them updated. 
This issue is with the build of the app in production mode which gives mesages like
>Dropping unreachable code [C:/Users/ahaan/Desktop/Angular2/angular-forms/~/style-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/~/css-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/~/postcss-loader!C:/Users/ahaan/Desktop/Angular2/angular-forms/src/styles.css:12,0]

**Comment by clydin** [comment]
What is the actual failure?

**Comment by filipesilva** [comment]
Those warnings should break your build, as @clydin mentions. There must be something else. Make sure to check the browser console as well.

**Comment by ahaan729** [comment]
@clydin I agree with you that those warnings are pretty much informative. 
I am just worried about the warning saying 
`WARNING in styles.3dd438f66d18a543d9d3.bundle.js from UglifyJs`
As the unreachable codes are the `bootstrap.css` and `styles.css` files. 

@filipesilva I checked the browser console it said the `.js files` could not be loaded.

I tried to rebuild it using the same command `ng build -p --bh /ng-forms/`. It gave the same warnings but after deploying it over the server it worked fine this time. (Clueless what happened earlier)

@filipesilva @clydin I was wondering, what it actually means when the warnings says `Dropping unreachable code` i.e., dropping the `.css` files.
Will that be a issue affecting the styles(look) that is implement on the app using bootstrap or custom css files?
